nuxeo-jsf/nuxeo-platform-webapp-base/src/main/resources/web/nuxeo.war/widgets/user_group_display_template.xhtml

```diff
@@ -100,5 +101,5 @@
     </h:commandLink>
   </c:if>
 </ui:fragment>
-<h:outputText rendered="#{empty info.type}" value="#{item}" />
+<h:outputText rendered="#{empty info.type and not empty data}" value="#{data}" />
```

Comment:
for backport, i still wonder if we'd better keep this param named "item" instead of "data"

Reply:
unless we redefine the param again and pass it through i guess, but let's forget about it